src/Components/Form.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+
+import InputItem from "../Components/InputItem";
+import MessageInput from "../Components/MessageInput";
+import SubmitButton from "../Components/SubmitButton";
+
+import Axios from "axios";
+
+class Form extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: "",
+      email: "",
+      message: "",
+      sent: false,
+    };
+  }
+
+  handleName = (e) => {
+    this.setState({
+      name: e.target.value,
+    });
+  };
+
+  handleEmail = (e) => {
+    this.setState({
+      email: e.target.value,
+    });
+  };
+
+  handleMessage = (e) => {
+    this.setState({
+      message: e.target.value,
+    });
+  };
+
+  formSubmit = (e) => {
+    e.preventDefault();
+
+    let data = {
+      name: this.state.name,
+      email: this.state.email,
+      message: this.state.message,
+    };
+
+    Axios.post("/api/forma", data)
+      .then((res) => {
+        this.setState(
+          {
+            sent: true,
+          },
+          this.resetForm()
+        );
+      })
+      .catch(() => {
```

Comment:
Would be useful to pass in `err` or `error` or `e` here and then in the console.log function, you can specify the error like so:
``` 
console.log(
`Message  not sent. Error: ${error}`)
```